cloudwatch_monitoring/sns.py

```diff
@@ -0,0 +1,184 @@
+"""
+module for creating sns widgets
+
+"""
+import boto3
+
+from .lookups import sns_topics
+from .constants import positioning
+
+
+def create_sns_widgets(region, deploy_stage):
+    """
+    Creates the list of SNS widgets.
+
+    :param region: Typically 'us-west-2'
+    :param deploy_stage: The deploy tier, DEV, TEST, QA, PROD-EXTERNAL
+    :return: list of SNS widgets
+    :rtype: list
+    """
+    sns_widgets = []
+
+    # set dimensions of the sns title widget
+    positioning['width'] = 24
```

Comment:
what are the units?  Can you add in comments?

Reply:
Good question! I have some doc written out on the nuances of the positioning object here in the code: https://github.com/usgs/aqts-capture-cloudwatch-dashboard/blob/main/cloudwatch_monitoring/constants.py#L6